Documentation/NextSteps/Index.rst

```diff
@@ -8,7 +8,7 @@ Next Steps and Further Reading
 ==============================
 
 All of TYPO3's documentation is available 
-`docs.typo3.org <https://docs.typo3.org/typo3cms/>`_.
+`docs.typo3.org <https://docs.typo3.org/>`_.
```

Comment:
Can you also update all other similar links here, too?

Reply:
maybe

Reply:
In this branch I can only find this one.